### PR TITLE
More specific GH pull request title

### PR DESCRIPTION
**Pull request by nvkelso**
When I browse PRs created by Write Field it's hard to parse the name of the place, because the casing in the PR title is all Tiltle Casing and `Place` looks like it's part of the place's name when it's not. 

Instead I'd like to know the feature's placetype (which is a proxy for how much effort the PR review will have based on it's downstream impact and scope), and that should be lowercase so it's easier to read the name part. To do this I add a new variable up top, and then modify the string substitution in the PR title creation.

Changes to:

- `Update country United States`
- `Update region New York`
- `Update locality Mill Valley`
- `Update neighbourhood Cole Valley`

From: 

- `Update Place United States`
- `Update Place New York`
- `Update Place Mill Valley`
- `Update Place Cole Valley`

@iandees I remember it was tricky to test the GH Pull Request part without a deploy. Can you help me with that part, please?

### CODE REVIEW

**Comment by iandees** [comment]
I don't think I ever got a decent dev/PR test setup working when it comes to interactions with Github. This is a pretty straightforward change, so I'm ok with merging it as-is.

**Comment by iandees** [comment]
I still have to manually deploy this, so I'll do that later this evening.

**Comment by iandees** [comment]
(It was deployed an hour ago)

**Comment by nvkelso** [comment]
Verified, thanks!
